service/src/main/resources/config/aou/test/SC2022Q4R6/original/SC2022Q4R6.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "SC2022Q4R6",
+  "displayName": " Controlled Tier Vxx",
+  "dataPointers": [
+    { "type": "BQ_DATASET",
+      "name": "omop_dataset",
+      "projectId": "all-of-us-ehr-dev",
+      "datasetId": "SC2022Q4R6",
+      "queryProjectId": "fc-aou-cdr-synth-test-2"
```

Comment:
Just curious, why do you need a different `queryProjectId` here than in the SR underlay?

Reply:
Because this is Controlled Tier, the other one is Registered Tier. Right now they are the sme but controlled tier for example will later have zip_socioeconomic, and other genomic tables....